caf-audit/src/main/java/com/hpe/caf/auditing/AuditConnectionFactory.java

```diff
@@ -22,8 +22,6 @@
 import com.hpe.caf.auditing.webserviceclient.WebServiceClientAuditConnection;
 import com.hpe.caf.auditing.webserviceclient.WebServiceClientException;
 
-import java.net.MalformedURLException;
```

Comment:
Spotted when I had viewed the AuditConnectionFactory during the write up. Can remove from this branch and make the change directly to develop